### PR TITLE
Devops/dividing docker compose

### PR DESCRIPTION
**Pull request by LouisHrg**
## Status
- [x] READY
- [ ] WORK IN PROGRESS

## Description
A few sentences describing the overall goals of the pull request's commits.

## Changes to the database (need to update DB)
- [ ] YES
- [x] NO

## Changes to the fixtures (need to re run fixture)
- [ ] YES
- [x] NO

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,23 +10,22 @@ install-docker-tools:
 
 install:
 	docker run --rm -v $$(pwd):/app composer install \
-	&& docker-compose exec php yarn install
+	&& docker-compose exec php yarn install \
+	&& docker-compose exec php yarn build \
+	&& docker-compose exec php make dev
```
Comment (Sylcab74) [inline Makefile:15]: I think that you should not use `docker-compose exec php yarn install` in the `install` but use your `make yarn` and `make yarn-build`

Comment (Sylcab74) [reply Makefile:15]: Same for `docker-compose exec php make dev` you could directly enter in the container in the `make dev` as in your `yarn-build` and `yarn`

Comment (LouisHrg) [reply Makefile:15]: Yeah but the server don't have any task runner or package manager installed, all these actions are done on the container

Comment (Sylcab74) [reply Makefile:15]: Yeah and your `make yarn-build` and `make yarn` are executing commands in your container

**Pull request merged**